### PR TITLE
Changes in `config.yml` template reflected on documentation

### PR DESCRIPTION
**Pull request by verdx**
## Description

This PR reflects on the documentation the changes for PR https://github.com/wazuh/wazuh-packages/pull/1901 done for issue https://github.com/wazuh/wazuh-packages/issues/1860

### CODE REVIEW

File: source/_templates/installations/indexer/common/generate_certificates.rst [hunk 1]
```diff
@@ -16,25 +16,36 @@
...
-         
+             #- name: node-2
+             #  ip: <indexer-node-ip>
+             #- name: node-3
+             #  ip: <indexer-node-ip>
+
            # Wazuh server nodes
-           # Use node_type only with more than one Wazuh manager
+           # If there is more than one Wazuh server 
+           # node each one must have a node_type
```
Comment (alberpilot) [inline source/_templates/installations/indexer/common/generate_certificates.rst:26]:
```suggestion
           # If there is more than one Wazuh server 
           # node, each one must have a node_type
```

File: source/_templates/installations/indexer/common/generate_certificates.rst [hunk 2]
```diff
@@ -16,25 +16,36 @@
...
-               ip: <dashboard-node-ip>
+             #  node_type: master
+             #- name: wazuh-2
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+             #- name: wazuh-3
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+
+
```
Comment (alberpilot) [inline source/_templates/installations/indexer/common/generate_certificates.rst:38]: An unnecessary empty line.

File: source/_templates/installations/indexer/common/generate_certificates.rst [hunk 3]
```diff
@@ -16,25 +16,36 @@
...
+          # Wazuh dashboard nodes
+          dashboard:
+            - name: dashboard
+              ip: <dashboard-node-ip>
+
+
```
Comment (alberpilot) [inline source/_templates/installations/indexer/common/generate_certificates.rst:44]: Same.

File: source/installation-guide/wazuh-indexer/installation-assistant.rst [hunk 4]
```diff
@@ -44,25 +44,36 @@ Indicate your deployment configuration, create the SSL certificates to encrypt c
...
-             #   ip: <indexer-node-ip>
-         
+             #- name: node-2
+             #  ip: <indexer-node-ip>
+             #- name: node-3
+             #  ip: <indexer-node-ip>
+
            # Wazuh server nodes
-           # Use node_type only with more than one Wazuh manager
+           # If there is more than one Wazuh server
```
Comment (alberpilot) [inline source/installation-guide/wazuh-indexer/installation-assistant.rst:53]: Reported

File: source/installation-guide/wazuh-indexer/installation-assistant.rst [hunk 5]
```diff
@@ -44,25 +44,36 @@ Indicate your deployment configuration, create the SSL certificates to encrypt c
...
-             - name: dashboard
-               ip: <dashboard-node-ip>
+             #  node_type: master
+             #- name: wazuh-2
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+             #- name: wazuh-3
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+
```
Comment (alberpilot) [inline source/installation-guide/wazuh-indexer/installation-assistant.rst:65]: Idem

File: source/installation-guide/wazuh-indexer/installation-assistant.rst [hunk 6]
```diff
@@ -44,25 +44,36 @@ Indicate your deployment configuration, create the SSL certificates to encrypt c
...
+
+          # Wazuh dashboard nodes
+          dashboard:
+            - name: dashboard
+              ip: <dashboard-node-ip>
+
+
```
Comment (alberpilot) [inline source/installation-guide/wazuh-indexer/installation-assistant.rst:72]: Idem

File: source/user-manual/certificates.rst [hunk 7]
```diff
@@ -37,30 +37,41 @@ To create the certificates, edit the ``config.yml`` file and replace the node na
...
+           indexer:
+             - name: node-1
+               ip: <indexer-node-ip>
+             #- name: node-2
+             #  ip: <indexer-node-ip>
+             #- name: node-3
+             #  ip: <indexer-node-ip>
+
+           # Wazuh server nodes
+           # If there is more than one Wazuh server
```
Comment (alberpilot) [inline source/user-manual/certificates.rst:51]: Reported

File: source/user-manual/certificates.rst [hunk 8]
```diff
@@ -37,30 +37,41 @@ To create the certificates, edit the ``config.yml`` file and replace the node na
...
+               ip: <wazuh-manager-ip>
+             #  node_type: master
+             #- name: wazuh-2
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+             #- name: wazuh-3
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+
+
```
Comment (alberpilot) [inline source/user-manual/certificates.rst:64]: Idem

File: source/user-manual/certificates.rst [hunk 9]
```diff
@@ -37,30 +37,41 @@ To create the certificates, edit the ``config.yml`` file and replace the node na
...
+          # Wazuh dashboard nodes
+          dashboard:
+            - name: dashboard
+              ip: <dashboard-node-ip>
+
+
```
Comment (alberpilot) [inline source/user-manual/certificates.rst:70]: Idem

File: source/_templates/installations/indexer/common/generate_certificates.rst (hunk 2 above)
Comment (verdx) [reply source/_templates/installations/indexer/common/generate_certificates.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/_templates/installations/indexer/common/generate_certificates.rst (hunk 3 above)
Comment (verdx) [reply source/_templates/installations/indexer/common/generate_certificates.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/installation-guide/wazuh-indexer/installation-assistant.rst (hunk 4 above)
Comment (verdx) [reply source/installation-guide/wazuh-indexer/installation-assistant.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/installation-guide/wazuh-indexer/installation-assistant.rst (hunk 5 above)
Comment (verdx) [reply source/installation-guide/wazuh-indexer/installation-assistant.rst:65]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/installation-guide/wazuh-indexer/installation-assistant.rst (hunk 6 above)
Comment (verdx) [reply source/installation-guide/wazuh-indexer/installation-assistant.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/user-manual/certificates.rst (hunk 7 above)
Comment (verdx) [reply source/user-manual/certificates.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/user-manual/certificates.rst (hunk 8 above)
Comment (verdx) [reply source/user-manual/certificates.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

File: source/user-manual/certificates.rst (hunk 9 above)
Comment (verdx) [reply source/user-manual/certificates.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607

**Review by alberpilot — APPROVED** [review]

File: source/_templates/installations/indexer/common/generate_certificates.rst [hunk 10]
```diff
@@ -16,25 +16,34 @@
...
+             #- name: wazuh-3
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+
+          # Wazuh dashboard nodes
+          dashboard:
+            - name: dashboard
+              ip: <dashboard-node-ip>
+
+          # Wazuh dashboard nodes
```
Comment (s-ocando) [inline source/_templates/installations/indexer/common/generate_certificates.rst:43]: The "Wazuh dashboard nodes" section is duplicated. This issue is present in all files.

Comment (verdx) [reply source/_templates/installations/indexer/common/generate_certificates.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/c9409cd3b85b4d6f32c4e1e30ad2bbb2de420d43

**Review by s-ocando — APPROVED** [review]

**Pull request merged**